README.md

```diff
@@ -156,15 +155,15 @@ rs.start_trace
 rs.stop_trace
 ```
 
-#### `Rotoscope#mark`
+#### `Rotoscope#mark(str = "")`
 
- Inserts a marker '---' to divide output. Useful for segmenting multiple blocks of code that are being profiled.
+ Inserts a marker '---' to divide output. Useful for segmenting multiple blocks of code that are being profiled. If `str` is provided, the line will be prefixed by '---', followed by the string passed.
```

Comment:
nitpick, but the line is prefixed by `--- `. If you include the space, people don't have to wonder whether to add it themselves (yeah, I know the example says this)